src/include/sof/common.h

```diff
@@ -83,12 +87,12 @@
 
 #endif
 
-/* This most basic ALIGN() must be used in header files that are
- * included in both C and assembly code. memory.h files require this
- * exact spelling matching the linker function because memory.h values
- * are _also_ copied unprocessed to the .x[.in] linker script
+/*
+ * memory.h files require this exact spelling of the ALIGN() macro, matching the
+ * linker function name because memory.h values are _also_ copied unprocessed to
+ * the .x[.in] linker script
```

Comment:
"unprocessed" was an exaggeration from me, can you please change to something like "because memory.h is also included in the .x[.in] linker script _without_ any ALIGN macro."